### PR TITLE
Storybook: Add intro and reorganize

### PR DESCRIPTION
**Pull request by tskarhed**
- Adds intro that is similar to the README.
- Reorganizes stories
- Hides addons panel for ThemeColors story

### CODE REVIEW

File: packages/grafana-ui/src/components/ColorPicker/ColorPicker.story.mdx [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 
-<Meta title="General/ColorPicker/ColorPicker"/>
+<Meta title="Pickers and Editors/ColorPicker/ColorPicker" />
 
 # ColorPicker
 
 The `ColorPicker` component group consists of several building blocks that are combined in Grafana to create the `ColorPicker`: popover, pickers and palettes. There are different combinations of these building blocks depending on where the `ColorPicker` is used in Grafana.
 
-The `Popover` is a tabbed view where you can switch between `Palettes`. The `NamedColorsPalette` shows an arrangement of  preset colors, while the `SpectrumPalette` is an unlimited HSB color picker. The preset colors are opimized to work well with both light and dark theme. `Popover` is triggered for example by the series legend of graphs, or by `Pickers`.
+The `Popover` is a tabbed view where you can switch between `Palettes`. The `NamedColorsPalette` shows an arrangement of preset colors, while the `SpectrumPalette` is an unlimited HSB color picker. The preset colors are opimized to work well with both light and dark theme. `Popover` is triggered for example by the series legend of graphs, or by `Pickers`.
```
Comment (Clarity-89) [inline packages/grafana-ui/src/components/ColorPicker/ColorPicker.story.mdx:9]:
```suggestion
The `Popover` is a tabbed view where you can switch between `Palettes`. The `NamedColorsPalette` shows an arrangement of preset colors, while the `SpectrumPalette` is an unlimited HSB color picker. The preset colors are opimized to work well with both light and dark theme. `Popover` is triggered, for example, by the series legend of graphs, or by `Pickers`.
```

**Review by Clarity-89 — APPROVED** [review]
Great job!

**Review by jessover9000 — APPROVED** [review]
Nice!

File: packages/grafana-ui/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Grafana UI components library
 
-> **@grafana/toolkit is currently in ALPHA**. Core API is unstable and can be a subject of breaking changes!
+> **@grafana/ui is currently in BETA**. Core API is unstable and can be subject to changes!
```
Comment (dprokop) [inline packages/grafana-ui/README.md:3]:
```suggestion
> **@grafana/ui is currently in BETA**. 
```

File: packages/grafana-ui/src/Intro.story.mdx [hunk 3]
```diff
@@ -1,5 +1,32 @@
-<Meta title="Docs Overview/Intro" parameters={{ options: { isToolshown: false}, id: 1}}/>
+<Meta title="Docs Overview/Intro" parameters={{ options: { isToolshown: false }, id: 1 }} />
 
-# @grafana/ui
+# @grafana/ui components library
 
-Welcome to @grafana/ui
+> **@grafana/ui is currently in BETA**. Core API is unstable and can be subject to changes!
```
Comment (dprokop) [inline packages/grafana-ui/src/Intro.story.mdx:5]:
```suggestion
> **@grafana/ui is currently in BETA**.
```

File: packages/grafana-ui/src/Intro.story.mdx [hunk 4]
```diff
@@ -1,5 +1,32 @@
...
+The `Canvas` tab can be used for:
+
+- Changing props for the component in the Knobs tab
+- Seeing code implementation in the Story tab
+
+The `Docs` tab provides:
+
+- Written docs with description on how to use the component
+- A props table
+- Code examples
```
Comment (dprokop) [inline packages/grafana-ui/src/Intro.story.mdx:32]: Great!


Can you add also section with links to grafana documentation and tutorials? Like here https://github.com/grafana/simple-react-panel

**Review by dprokop — APPROVED** [review]

File: packages/grafana-ui/src/Intro.story.mdx (hunk 4 above)
Comment (tskarhed) [reply packages/grafana-ui/src/Intro.story.mdx:32]: Awesome! Also got some feedback from @marcusolsson that I'm going to implement

**Pull request merged**